core/model.py

```diff
@@ -58,26 +57,47 @@ def _load_assets(self, path):
         self.proc_fn = get_processing_word(vocab_words, vocab_chars, lowercase=True, chars=True)
 
         self.id_to_tag = {idx: v for v, idx in vocab_tags.items()}
+        self.dict_vocal_tags_len = len(vocab_tags)
+        self.id_to_tag.update({self.dict_vocal_tags_len: 'O'})
```

Comment:
Is this really required?

I guess its for the `_post_process` to only return the predicted tags for input words, not for padding?

You should be able to use `pad_tag` to filter?

Reply:
I was getting an error when using the V1  dataset as prediction number had `pad_tag`. I have code to update `id_to_tag` as well as remove tag during the post-processing step.